### ISSUE TITLE
make a self-balancing robot will be cool

### ISSUE THREAD

**Issue by Baurito** [EmanuelFeru/hoverboard-sideboard-hack-GD#6]
Theoretically, you can make a self-balancing robot. This will be your cool project. For example, like Ninebot mini

**Comment by EFeru** [comment]
I agree.. :) I am investigating it. However, for the moment my time is limited, so it goes slow.

**Comment by AlexJenjer** [comment]
https://www.youtube.com/watch?v=sZHZg2eifek
